### PR TITLE
Woche7

### PR DESCRIPTION
**Pull request by Koch-Dustin**
Changes:
   Add fiftyville
   Add movies
   Add songs

PR: #9

### CODE REVIEW

File: movies/11.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+SELECT movies.title FROM people JOIN stars ON people.id = stars.person_id JOIN movies ON stars.movie_id = movies.id JOIN ratings ON movies.id = ratings.movie_id WHERE people.name = "Chadwick Boseman" ORDER BY rating DESC LIMIT 5;
```
Comment (FrederikFricke) [inline movies/11.sql:1]:
```suggestion
SELECT movies.title
    FROM people 
    JOIN stars 
       ON people.id = stars.person_id 
    JOIN movies 
       ON stars.movie_id = movies.id 
    JOIN ratings
       ON movies.id = ratings.movie_id
       WHERE people.name = "Chadwick Boseman"
       ORDER BY rating DESC LIMIT 5;
    
```

stick to the convention you used on the `fiftyville/log.sql` file. this is also for the other `.sql` files.

**Review by FrederikFricke — CHANGES_REQUESTED** [review]

File: fiftyville/log.sql [hunk 2]
```diff
@@ -0,0 +1,119 @@
+-- Infos zum Diebstahl nachschauen. Bei der gegeben Zeit und Ort.
+SELECT description FROM crime_scene_reports WHERE year = 2021 AND month = 7 AND day = 28 AND street = 'Humphrey Street';
+
```
Comment (DevBastiS) [inline fiftyville/log.sql:3]:
```suggestion
```

File: fiftyville/log.sql [hunk 3]
```diff
@@ -0,0 +1,119 @@
+-- Infos zum Diebstahl nachschauen. Bei der gegeben Zeit und Ort.
+SELECT description FROM crime_scene_reports WHERE year = 2021 AND month = 7 AND day = 28 AND street = 'Humphrey Street';
```
Comment (DevBastiS) [inline fiftyville/log.sql:2]:
```suggestion
SELECT description FROM crime_scene_reports
WHERE year = 2021 AND month = 7
  AND day = 28
  AND street = 'Humphrey Street';
```

File: fiftyville/log.sql [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+-- Drei zeugen gefunden, einer erwähnt am 28. July. 2021 "Bäcker"
+SELECT name, transcript
+  FROM interviews
+ WHERE year = 2021
+   AND month = 7
+   AND day = 28
+   AND transcript LIKE '%bakery%'
+ ORDER BY name;
+ -- Zeugen: Eugene, Raymond und Ruth
+
```
Comment (DevBastiS) [inline fiftyville/log.sql:26]:
```suggestion
```

File: fiftyville/log.sql [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+-- Gucken wer transaktionen gemacht hat
+SELECT account_number, amount
+  FROM atm_transactions
+ WHERE year = 2021
+   AND month = 7
+   AND day = 28
+   AND atm_location = 'Leggett Street'
+   AND transaction_type = 'withdraw';
```
Comment (DevBastiS) [inline fiftyville/log.sql:35]:
```suggestion
   AND transaction_type = 'withdraw';
   
```

File: fiftyville/log.sql [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+  JOIN bank_accounts
+    ON people.id = bank_accounts.person_id
+  JOIN atm_transactions
+    ON bank_accounts.account_number = atm_transactions.account_number
+ WHERE atm_transactions.year = 2021
+   AND atm_transactions.month = 7
+   AND atm_transactions.day = 28
+   AND atm_transactions.atm_location = 'Leggett Street'
+   AND atm_transactions.transaction_type = 'withdraw';
+
```
Comment (DevBastiS) [inline fiftyville/log.sql:48]:
```suggestion
```

File: fiftyville/log.sql [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+
+-- Raymond hört gespräch um ein Ticket für eine Flugbuchung für den frühsten Flug am July 29, 2021
```
Comment (DevBastiS) [inline fiftyville/log.sql:50]: Wenn schon Kommentare dann bitte richtiges deutsch, sonst die Kommentare entfernen bitte.

File: fiftyville/log.sql [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+  JOIN flights
+    ON airports.id = flights.destination_airport_id
+ WHERE flights.origin_airport_id =
+       (SELECT id
+          FROM airports
+         WHERE city = 'Fiftyville')
+   AND flights.year = 2021
+   AND flights.month = 7
+   AND flights.day = 29
+ ORDER BY flights.hour, flights.minute;
```
Comment (DevBastiS) [inline fiftyville/log.sql:67]:
```suggestion
 ORDER BY flights.hour, flights.minute;
 
```

File: fiftyville/log.sql [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+  JOIN passengers
+    ON people.passport_number = passengers.passport_number
+  JOIN flights
+    ON passengers.flight_id = flights.id
+ WHERE flights.year = 2021
+   AND flights.month = 7
+   AND flights.day = 29
+   AND flights.hour = 8
+   AND flights.minute = 20
+ ORDER BY passengers.passport_number;
```
Comment (DevBastiS) [inline fiftyville/log.sql:81]:
```suggestion
 ORDER BY passengers.passport_number;
 
```

File: fiftyville/log.sql [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+-- Mögliche Namen der Anrufer herausfinden und in die "Suspect List" tun Sortieren nach anruf dauer
+SELECT name, phone_calls.duration
+  FROM people
+  JOIN phone_calls
+    ON people.phone_number = phone_calls.caller
+ WHERE phone_calls.year = 2021
+   AND phone_calls.month = 7
+   AND phone_calls.day = 28
+   AND phone_calls.duration <= 60
+ ORDER BY phone_calls.duration;
```
Comment (DevBastiS) [inline fiftyville/log.sql:92]:
```suggestion
 ORDER BY phone_calls.duration;
 
```

File: fiftyville/log.sql [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+SELECT name, phone_calls.duration
+  FROM people
+  JOIN phone_calls
+    ON people.phone_number = phone_calls.receiver
+ WHERE phone_calls.year = 2021
+   AND phone_calls.month = 7
+   AND phone_calls.day = 28
+   AND phone_calls.duration <= 60
+   ORDER BY phone_calls.duration;
+
```
Comment (DevBastiS) [inline fiftyville/log.sql:103]:
```suggestion
```

File: movies/1.sql [hunk 12]
```diff
@@ -0,0 +1,2 @@
+SELECT title FROM movies
+WHERE year = 2008;
```
Comment (DevBastiS) [inline movies/1.sql:2]: Missing EOF in many files.

File: songs/3.sql [hunk 13]
```diff
@@ -0,0 +1 @@
+SELECT name FROM songs ORDER BY duration_ms DESC LIMIT 5;
```
Comment (DevBastiS) [inline songs/3.sql:1]: Structure this queries like you did with the others.
```suggestion
SELECT name FROM songs
ORDER BY duration_ms DESC
LIMIT 5;
```

**Review by DevBastiS — CHANGES_REQUESTED** [review]

**Review by stefanluth — CHANGES_REQUESTED** [review]
Add EOFs

File: songs/1.sql [hunk 14]
```diff
@@ -0,0 +1 @@
+SELECT title FROM movies WHERE year = 2008;
```
Comment (DevBastiS) [inline songs/1.sql:1]: Missing EOF in many files

File: songs/3.sql (hunk 13 above)
Comment (DevBastiS) [reply songs/3.sql:1]: Still open.

**Review by DevBastiS — CHANGES_REQUESTED** [review]

**Review by stefanluth — CHANGES_REQUESTED** [review]
Choose one code style

Screenshot 2022-09-20 at 09 37 20
Screenshot 2022-09-20 at 09 37 17

**Review by DevBastiS — APPROVED** [review]
What Stefan said.

**Review by stefanluth — CHANGES_REQUESTED** [review]
Indentation is still off :)

File: fiftyville/log.sql [hunk 15]
```diff
@@ -0,0 +1,124 @@
...
+-- Drei zeugen gefunden, einer erwähnt am 28. July. 2021 "Bäcker"
+SELECT name, transcript
+    FROM interviews
+    WHERE year = 2021
+        AND month = 7
+        AND day = 28
+        AND transcript LIKE '%bakery%'
+    ORDER BY name;
+ -- Zeugen: Eugene, Raymond und Ruth
```
Comment (stefanluth) [inline fiftyville/log.sql:29]:
```suggestion
-- Zeugen: Eugene, Raymond und Ruth
```

File: fiftyville/log.sql [hunk 16]
```diff
@@ -0,0 +1,124 @@
...
+        AND atm_transactions.day = 28
+        AND atm_transactions.atm_location = 'Leggett Street'
+        AND atm_transactions.transaction_type = 'withdraw';
+
+-- Raymond hört ein gespräch wo es um ein Ticket für den frühsten Flug am July 29, 2021 geht.
+-- Den landeort finden würde bedeuten den Fluchtort zu finden
+SELECT abbreviation, full_name, city
+    FROM airports
+    WHERE city = 'Fiftyville';
+ -- Flüge vom 29. July überprüfen und nach Zeit sortieren
```
Comment (stefanluth) [inline fiftyville/log.sql:58]:
```suggestion
-- Flüge vom 29. July überprüfen und nach Zeit sortieren
```

**Review by stefanluth — COMMENTED** [review]
:)

**Review by stefanluth — APPROVED** [review]

**Pull request merged**